assets/scss/bootstrap-migration/custom/_backgrounds.scss

```diff
@@ -0,0 +1,6 @@
+//Last argument in bg-variant is set to 'true' to prevent warning about bg-variant being deprecated
+//The official @mixin for bg-variant and bg-gradient-variant in bootstrap 4.5.0 is now @mixin bg-variant($parent, $color, $ignore-warning: false)
+//But in official docs this mixin is still presented and there's no alternatives right now.
+@each $color, $value in $theme-colors {
+  @include bg-variant('.bg-#{$color}-transparent', rgba($value, 0.5), true);
```

Comment:
I would suggest `semitransparent` instead, because transparent is 0% alpha channel

Reply:
done